apps/tests/lib/embedding_tests.ex

```diff
@@ -174,12 +175,7 @@ defmodule Tests.EmbeddingTests do
     }
 
     json_string = Jason.encode!(content)
-    filename = "#{@output_path}/#{generate_output_name()}.json"
+    filename = "#{@output_path}/#{TestUtils.generate_output_name()}.json"

```

Comment:
Consider using `Path.join` so that it is portable

Reply:
done